UNITTESTS/fakes/drivers/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
```

Comment:
I did some tests and you can make this simpler: no includes required. The macros, like functions are project scoped.

Reply:
As discussed in offline chat, please push your changes related to make a macro to project scope